dao-user-reference.adoc

```diff
@@ -0,0 +1,192 @@
+
+1. **Every DAO action is an on-chain Bitcoin transaction.** Every DAO action requires BSQ, and since BSQ is bitcoin, every DAO action is also recorded on the Bitcoin blockchain. As noted above, this does NOT mean you should try to form your own BSQ transactions. But it does mean that every action you take on the Bisq DAO is immortalized on the Bitcoin blockchain, publicly verifiable, and virtually impossible to reverse.
+
+2. **Bitcoin provides transaction integrity, and Bisq's own P2P network provides storage.** DAO actions such as making proposals and voting include extra data such as text and links that don't belong on the Bitcoin blockchain. This extra data is stored on Bisq's own P2P network (literally, Bisq users' hard drives). A hash of that extra data is included in the `OP_RETURN` output of the corresponding Bitcoin transaction. Then, the Bitcoin transaction ID is added to the extra data and broadcast to peers on the Bisq network.
+
+You can visualize it like this:
+
+.How Bisq stores data
+image::bisq-dao-data-model.png[How Bisq stores data]
+
+Storing a hash of the extra data on the Bitcoin blockchain ensures any attempts to manipulate it will fail, since the hash of the manipulated data will change and then not match the hash recorded in the corresponding Bitcoin transaction.
+
+NOTE: There's a bit of a chicken-and-egg problem here, since the Bitcoin transaction ID can only be known after the Bitcoin transaction is made, and the hash of a data object can only be made once the Bitcoin transaction ID is known. To get around this, the data object is hashed with a blank string in place of the Bitcoin transaction ID, and the Bitcoin transaction ID is added afterward. Later, when the integrity of the data object needs to be validated, the Bitcoin transaction ID is replaced with a blank string before the data object is hashed.
+
+=== Voting Cycle
+
+Voting cycles are timed to last about 1 month. Duration is measured in blocks, so actual times may vary.
+
+==== Proposal Phase
+
+_3600 blocks, about 24 days_
+
+During this time, Bisq stakeholders (contributors and traders) can make proposals. It is not possible to edit a proposal, but it is possible to remove a proposal and submit a corrected one (only during this phase).
+
+There is a small fee to make proposals to discourage spam. Fees for proposals that are removed are not reimbursed.
+
+Proposals requesting the issuance of new BSQ will also require a small amount of BTC to be included in the proposal transaction so there is BTC to 'color' into BSQ. The spec requires 100 satoshis per BSQ token, so a request for 1000 BSQ will require 1000 × 100 = 100,000 satoshis.
+
+Keep in mind that the actual BSQ you end up with will be net the proposal fee: if your request for 1000 BSQ is approved, and if the proposal fee is 2 BSQ, you will end up with 998 BSQ in your wallet.
+
+For compensation requests, please make sure you follow <<compensation#, established conventions and recommendations>>.
```

Comment:
"<<compensation#, established conventions and recommendations>>"  is compensation# intended?

Reply:
Yes, AsciiDoc requires the `#` for links to external doc files. Without the `#`, it assumes the link is on the same page.

It's very odd...the opposite of what you would expect.